apps/bundle-size/src/index.js

```diff
@@ -0,0 +1,12 @@
+// @ts-check
```

Comment:
**thought:** we could leverage runtime type removal that I've prepared for babel-make-styles and friends to not need to run build before executing commands. if you'd be interested, lemme know and we can pair on that :) 

-> thus having whole implementation in TypeScript

Reply:
As this package has no dependencies we can can keep in this JS for now. I don't have issues there 🙄